pkg/kv/kvnemesis/operations.go

```diff
@@ -535,6 +539,11 @@ func (op RestartNodeOperation) format(w *strings.Builder, fctx formatCtx) {
 	op.Result.format(w)
 }
 
+func (op CrashNodeOperation) format(w *strings.Builder, fctx formatCtx) {
+	fmt.Fprintf(w, `env.Restarter.CrashServer(%d)`, int(op.NodeId))
```

Comment:
Should this and above 2 methods print `ServerController` instead of `Restarter`? If so, do the other 2 in the first commit which does the rename.